package.json

```diff
@@ -4,8 +4,8 @@
   "description": "CLI tool to run BigCommerce Stores locally for theme development.",
   "main": "index.js",
   "engines": {
-    "node": ">=14.0.0 <15.0.0",
-    "npm": ">=v6.11.3 <9.0.0"
+    "node": ">=17.0.0 <19.0.0",
```

Comment:
@jairo-bc your screen says that you've managed to run stencil on Node 14. Is that ok taking into account this line?

Reply:
updated